telemetry/otel.go

```diff
@@ -108,6 +108,10 @@ func (o *OtelClient) SetValidatorStatus(isVal bool) {
 	o.vi.IsValidator = isVal
 }
 
+func (o *OtelClient) IsValidator() bool {
+	return o.vi.IsValidator
+}
+
 func (o *OtelClient) GetValAddr() crypto.Address {
 	return o.vi.Address
```

Comment:
I feel like this should return an error if we are not a validator ?

Reply:
i don't think that really benefits the usage much. the address is always set because sdk/comet will always provision a node key. then its up to `x/telemetry` to use this address to determine whether we are a val or not. and you can still have a val address without being bonded.